frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/OpenMacheteTabAction.java

```diff
@@ -0,0 +1,40 @@
+package com.virtuslab.gitmachete.frontend.actions.base;
```

Comment:
Hmm I wouldn't put this to `base` package since this class is not meant to be a shared base for other actions... maybe add a `vcsmenu` (?) package